### DISCUSSION TITLE
No GStreamer support in 2.0.x binary releases

### DISCUSSION THREAD

**Discussion by itanswers** [savonet/liquidsoap#2497] [User Support]
There is no GStreamer support in 2.0.x binary releases. At least in deb-packages. Last binary release with GStreamer support was 1.4.4. 
Why did you drop GStreamer from binary releases?

OS: Ubuntu 20.04 LTS (Focal)

**Comment by toots** [comment]
gstreamer support is deprecated. We suggest trying to use the `ffmpeg` encoder and operators instead. Most functionalities should be available there now and we will actively respond to feature and bug reports with it.
